src/main/java/io/github/perrymant/moneymaker/FileIO.java

```diff
@@ -0,0 +1,31 @@
+package io.github.perrymant.moneymaker;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+class FileIO {
+
+    static String read(String fileName) throws IOException {
```

Comment:
No need for this to be `static`. In fact, other than `main`, try and avoid `static` unless you have a strong reason to use it.